build/js/live-editor.output_pjs.js

```diff
@@ -1213,15 +1213,27 @@ window.PJSOutput = Backbone.View.extend({
         this.config = options.config;
         this.output = options.output;
 
-        this.tester = new PJSTester(_.extend(options, {
-            workerFile: "pjs/test-worker.js"
-        }));
-
         this.render();
         this.bind();
- 
+
         this.build(this.$canvas[0]);
 
+        // get URL params
+        var params = {};
+        location.search.substring(1).split("&").forEach(function(param) {
+            var parts = param.split("=");
+            params[parts[0]] = decodeURIComponent(parts[1]);
+        });
+
+        // Initialize workers:
+        // The reason why workersDir is passed as a URL parameter is so 
+        // that pjs-output.js can do a GET request for the worker code and
+        // their dependencies without having to wait for the initial message
+        // from the parent frame that contains all of the paths.
```

Comment:
In the original version of this feature I injected the path directly into output.html.  I don't know what the path is on the server and so I'm using the ```workersDir``` path the live-editor.js has.  This was the most direct approach I could think of.